rag/advanced_rag/harness/tools/navigation.py

```
@@ -1023,3 +1032,873 @@ def _add_entities(new: list[dict], scope_key: str = "") -> list[str]:
+                        [],
+                        {"id": list(wanted)[:128]},
+                        [],
+                        None,
+                        0,
+                        128,
+                        index,
+                        [kb_id],
+                    )
+                    rows = settings.docStoreConn.get_fields(res, fields) or {}
+                except Exception:
+                    continue
```

Ruff (0.16.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)